pkg/tenant/db/topics.go

```diff
@@ -13,12 +14,15 @@ import (
 const TopicNamespace = "topics"
 
 type Topic struct {
-	OrgID     ulid.ULID `msgpack:"org_id"`
-	ProjectID ulid.ULID `msgpack:"project_id"`
-	ID        ulid.ULID `msgpack:"id"`
-	Name      string    `msgpack:"name"`
-	Created   time.Time `msgpack:"created"`
-	Modified  time.Time `msgpack:"modified"`
+	OrgID               ulid.ULID                `msgpack:"org_id"`
+	ProjectID           ulid.ULID                `msgpack:"project_id"`
+	ID                  ulid.ULID                `msgpack:"id"`
+	Name                string                   `msgpack:"name"`
+	State               pb.TopicTombstone_Status `msgpack:"state"`
+	ConfirmDeleteToken  string                   `msgpack:"confirm_delete_token"`
+	ConfirmDeleteExpire time.Time                `msgpack:"confirm_delete_expire"`
```

Comment:
I'd recommend the token be a base64 encoded msgpack data structure that includes the topic ID and expiration time stamp as well as a cryptographically randomly generated string. That way the token is self composed and in the future we can do things like cryptographically sign the token for further security.